qiskit/dagcircuit/dagcircuit.py

```diff
@@ -310,8 +314,7 @@ def _add_wire(self, wire):
             self.output_map[wire] = outp_node
             self._multi_graph.add_edge(inp_node._node_id,
                                        outp_node._node_id,
-                                       {'name': wire_name,
-                                        'wire': wire})
+                                       {'wire': wire})
```

Comment:
Do you think it would make sense to drop the dict wrapper here and just attach the bit/wire on the edge directly? We'd probably have to change some access patters from `edge['wire']` to `edge` not sure how much that is used.

Reply:
That's a good idea, it seems like only a handful of places check this property, so it seems a good time to remove the unneeded `dict`.

Reply:
Added in 664485e5dc .

Reply:
After starting https://github.com/Qiskit/retworkx/pull/252 I expect it was originally a dict because networkx only lets you weight edges with a dict of attributes instead of an object. After we switched to retworkx there wasn't really a reason to change it until now.